### PR TITLE
Finalize Cosmos DB Admin implementation

### PR DESCRIPTION
**Pull request by Torch3333**
This PR adds not yet implemented methods in the Cosmos DB Admin class.
Please have a look, thank you.

### CODE REVIEW

**Comment by Torch3333** [comment]
The failed CI is caused by a failure to install `cqlsh`. I am looking into it.

**Comment by Torch3333** [comment]
> The failed CI is caused by a failure to install `cqlsh`. I am looking into it.

The CI is fixed by https://github.com/scalar-labs/scalardb/pull/292

File: core/src/main/resources/cosmosdb_stored_procedure/mutate.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+function mutate(num, ...args) {
```
Comment (feeblefakie) [inline core/src/main/resources/cosmosdb_stored_procedure/mutate.js:1]: Nice 👍  It should be managed in core.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**